pkg/test/framework/components/istio/operator.go

```diff
@@ -222,10 +239,24 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 	for k, v := range cfg.Values {
 		installSettings = append(installSettings, "--set", fmt.Sprintf("values.%s=%s", k, v))
 	}
+
 	if c.environment.IsMulticluster() {
 		// Set the clusterName for the local cluster.
-		// This MUST match the clusterName in the remote secret for this cluster.
+		// This MUST match the clusterName in the remote secret for this cluster for multicluster to work.
 		installSettings = append(installSettings, "--set", "values.global.multiCluster.clusterName="+cluster.Name())
```

Comment:
For whatever reason, setting the clusterName for non-multicluster caused `TestAsymmetricMeshNetworkWithGatewayIP` to fail,  still trying to understand why.